coralproject/talk
# [CORL-443] Add user drawer to the community section of the moderation area

## What does this PR do?

Adds user drawer functionality when clicking on usernames in the community area.

## How do I test this PR?

- Head to the community tab in the moderation area
- Click on a user name in the users table
- User drawer should pop up
